### PR TITLE
Restructure sample to use separate commands

### PR DESCRIPTION
**Pull request by LandryNorris**
Use separate commands for various tasks (list, led, motor, servo, etc).

### CODE REVIEW

File: packages/sample/src/HubStringify.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { RevHub } from "@rev-robotics/expansion-hub";
+
+export async function toString(hub: RevHub): Promise<string> {
```
Comment (NoahAndrews) [inline packages/sample/src/HubStringify.ts:3]: In Typescript and Javascript, there can only be one function implementation with a given name in scope in a given file. Any file that imports this function can't have its own `toString()` function, so this unfortunately needs a more descriptive name.

Comment (NoahAndrews) [reply packages/sample/src/HubStringify.ts:3]: Also, why is this function async?

**Review by NoahAndrews — CHANGES_REQUESTED** [review]

File: packages/sample/src/HubStringify.ts (hunk 1 above)
Comment (NoahAndrews) [reply packages/sample/src/HubStringify.ts:3]: > This code is in the sample. It should never be imported by end users.

I know, but it's still a flaw within the sample codebase IMO, just one that doesn't matter as much

> getModuleAddress was originally async if I remember correctly. I think we can remove the async now.

Please do so

Comment (LandryNorris) [reply packages/sample/src/HubStringify.ts]: Resolved both

**Pull request merged**